types/node/index.d.ts

```diff
@@ -2329,32 +2331,31 @@ declare module "child_process" {
 
     export interface ForkOptions {
         cwd?: string;
-        env?: any;
+        env?: NodeJS.ProcessEnv;
         execPath?: string;
         execArgv?: string[];
         silent?: boolean;
-        stdio?: any[];
+        stdio?: StdioOptions;
+        windowsVerbatimArguments?: boolean;
         uid?: number;
         gid?: number;
-        windowsVerbatimArguments?: boolean;
     }
     export function fork(modulePath: string, args?: ReadonlyArray<string>, options?: ForkOptions): ChildProcess;
 
     export interface SpawnSyncOptions {
-        argv0?: string;
```

Comment:
I'm quite sure that `argv0` argument is also supported for sync even if it is missing in node docs.

Reply:
It does seem pretty unlikely that `spawnsync` wouldn't end up reusing most of the logic from `spawn` (or vice versa).  I'll put the option back here and see about getting the Node docs fixed -- if it turns out you really can't, I'll open a separate PR to take it out, I guess.

Reply:
That didn't take long: [found it](https://github.com/nodejs/node/blob/master/lib/child_process.js#L490).  Definitely uses the same logic for both functions.

Reply:
[Node PR](https://github.com/nodejs/node/pull/22231) is up.